app/src/cambodia/java/org/eyeseetea/malariacare/strategies/SettingsActivityStrategy.java

```diff
@@ -2,80 +2,53 @@
 
 import android.content.Intent;
 import android.preference.Preference;
+import android.preference.PreferenceCategory;
 import android.preference.PreferenceScreen;
-import android.support.v7.app.AlertDialog;
+import android.util.Log;
 
-import com.squareup.otto.Subscribe;
-
-import org.eyeseetea.malariacare.ProgressActivity;
+import org.eyeseetea.malariacare.LoginActivity;
 import org.eyeseetea.malariacare.R;
 import org.eyeseetea.malariacare.SettingsActivity;
-import org.eyeseetea.malariacare.data.sync.exporter.PushController;
-import org.eyeseetea.malariacare.layout.listeners.LoginRequiredOnPreferenceClickListener;
```

Comment:
@idelcano LoginRequiredOnPreferenceClickListener now is unused class, can you remove it?